test/TestAssets/TestProjects/NuGetConfigRandomPackageSources/NuGet.config

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
-    <add key="invalid_source" value="https://api.nuget.org/v3/invalid.json" />
-    <add key="invalid_source_2" value="https://api.nuget.org/v3/invalid-2.json" />
+    <clear />
+    <add key="example_source" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
```

Comment:
don't you need an invalid source as well? Looks like maybe the test itself now handles that by adding a duplicate.

Reply:
No, this was copied over from a similar test I did, but this one just checks to see if the source feed is not added twice (through the nuget.config and --add-source). I forgot to remove the second source and the name was unaccurate

Reply:
That also seems to be handled by `_nugetConfigWithInvalidSources`

Reply:
I thought the same thing at first!

Reply:
> That also seems to be handled by `_nugetConfigWithInvalidSources` in `WhenPassingIgnoreFailedSourcesItShouldNotThrow`

Yes. I might create a TestAsset for that in the future as IMO it looks cleaner than just having the string contents on the file